### PR TITLE
Add Google group doc.

### PR DESCRIPTION
**Pull request by garrypolley**
This adds the Google group documentation to the readme. Should resolve issue #9

clickable link to the google group: https://groups.google.com/forum/#!forum/canadarm

### CODE REVIEW

**Comment by garrypolley** [comment]
@gneatgeek @sambao21  I plan to merge this within the next day or so unless I get some feedback to change up the verbiage or link.

**Comment by sambao21** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Canadarm makes JavaScript logging easy and seamless, helping you figure out what
 Named for the [Canadarm](http://en.wikipedia.org/wiki/Canadarm) we hope this too helps you piece together
 log information to figure out what went wrong with your scripts.
 
+If you have any ideas or questions reach out to us on our [Google group][google-group-url].
```
Comment (gneatgeek) [inline README.md]: I think a comma is appropriate between questions and reach, but it's probably fine as-is if you want to leave it alone.

**Comment by gneatgeek** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (garrypolley) [inline README.md]: I'm not going to worry too much about it. In what I'm reading the "If you have any ideas or questions" is not an independent clause so the comma is not needed. Nick Overfield may have thoughts. 

Thanks for reviewing, I'll give it a merge.

**Pull request merged**